lips/lip-5.md

```diff
@@ -0,0 +1,158 @@
+
+
+# Specification
+
+## Account identifiers
+For communicating account identity, we propose using a compact, versioned and case-insensitive identifier. To meet this criteria, we selected the Bech32 encoding implementation used in Bitcoin Segwit (BIP 0173) excluding the Segwit byte known-length restrictions.
+
+### Desired attributes
+- Consistent - Users can build a muscle memory for identifying and using these account addresses
+- Atomic - The string identifier feels like a single unit. Users shouldn’t try to separate or truncate the string
+- Versioned - The format contains human readable information about how to interpret the payload, preventing subtle errors and reducing risk of linkability for human readable addresses
+- Error detecting - Bech32 checksums help clients validate input and reduce risk of bad transactions due to mistypings and truncations
+
+### Constraints for subaddresses
+- Fixed length at 8 bytes (we don’t accept other sizes, strictly 8 bytes).
+- Unique to their custodian/on-chain account: addresses must be non-ambiguous (ie. Recipient VASPs should not issue the same address to multiple users).
+- By convention, 8 zero bytes (`0x0000000000000000`) is reserved to denote the root (VASP owned) account.
+
+### Format
+The Libra Account Identifier consists of 
+* A prefix (also known as hrp (human readable part) which identifies the network version this address is intended for
+  * “lbr” for Mainnet addresses
+  * “tlb” for Testnet addresses
+* A Bech32 delimiter
+  * The character “1” (one)
+* A Bech32 version identifier
+  * The character “p” (version = 1) for on-chain with subaddress
+* A Bech32 encoded payload
+  * For version 1 is Libra account address + subaddress (16 + 8 bytes)
+The last 6 characters correspond to the Bech32 checksum
+
+Overall address format: *prefix* | *delimiter* | *version* | *encoded payload* | *checksum*
+
+### Example with explicit subaddress 
+Identifier information
+* Prefix (string)
+  * Network: `lbr`
+* Address type (version prefix): `01` (letter p in Bech32 alphabet)
+* Payload (in hex)
+  * Address: `f72589b71ff4f8d139674a3f7369c69b`
+  * Subaddress: `cf64428bdeb62af2`
+
+**Result**: `lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t`
+
+### Example without explicit subaddress (root account)
+Identifier information
+* Prefix (string)
+  * Network: `lbr`
+* Address type (version prefix): `01` (letter p in Bech32 alphabet)
+* Payload (in hex)
+  * Address: `f72589b71ff4f8d139674a3f7369c69b`
+  * Subaddress: `0000000000000000`
+
+**Result**: `lbr1p7ujcndcl7nudzwt8fglhx6wxnvqqqqqqqqqqqqqflf8ma`
+
+### Looking ahead
+In the future, we plan to define additional Account Identifier versions to support other forms of identity, such as a more human-readable subaddress format. These would leverage a similar overall structure but would have a different version identifier, preventing naming collisions.  
+
+## Intent identifiers
```

Comment:
Now that I read this again, I'm starting to think that we should perhaps split out intents from the account identifier LIP.  It feels like the first topic is over and then we start a new topic abruptly.  Thoughts?

Reply:
I agree that the transition is a bit abrupt (albeit hinted at the start). I do think that since these proposals are so closely related, a single LIP may be useful for centralizing discussion